packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Added favicon to webhelp [#801] (https://github.com/zowe/zowe-cli/issues/801).
```

Comment:
- Please add missing articles: "Added **a** favicon to **the** ... ."
- Use **Web Help** as two capitalized words since this explanatory text (as opposed to code).
- Not necessary, but might be nice to consider adding a reason for the favicon and/or explanation of where to expect the favicon: "Added a favicon to the Web Help **that displays in address bars, browser tabs, and WHEREELSE**."